docs/containers/debug-node.md

```diff
@@ -1,44 +1,22 @@
 DateApproved: 12/12/2019
-MetaDescription: Debug a Node.js app running in a Docker container, using Visual Studio Code.
+MetaDescription: How to configure and troubleshoot debugging of Node.js apps running in a Docker container, using Visual Studio Code.
 ---
-# Debugging Node.js within Docker containers
 
-## Walkthrough
+# Common Node.js Debugging Configuration Tasks
 
-Start with an existing Node.js application or create a new one
+When adding Docker files to a Node.js project, tasks and launch configurations are added intended to enable one to debug that application within a Docker container. However, due to the large ecosystem surrounding Node.js, those tasks cannot accommodate every application framework or library, which means that some applications will require additional configuration.
 
-```bash
-> express my-express-app
-> cd my-express-app
-> npm install
-> code .
-```
-
-Add Docker files to the application
-
-Press `kb(workbench.action.showCommands)` to open the command palette and select the `Docker: Add Docker Files to Workspace` command
-
-Select `Node.js`
-
-Select either `Yes` or `No` to include Docker Compose files. (Neither option affects whether you can debug the application in VS Code.)
-
-Enter the port on which the application listens (e.g. typically `3000`)
-
-The Docker extension will add, at minimum, a `.dockerignore` and `Dockerfile` files. It will also add tasks for building and running the application within a Docker container, as well as a launch configuration for debugging the application.
-
-## Common Node.js Debugging Configuration Tasks
-
-### Configuring the Docker container entry point
+## Configuring the Docker container entry point
 
 The Docker extension infers the entry point of the Docker container--that is, the command line for starting the application in a debug mode within the Docker container--via properties of `package.json`.  The extension will first look for the `start` script in the `scripts` object; if found and, if it starts with a `node` or `nodejs` command, it will be used to build the command line for starting the application in debug mode.  If not found or, if not a recognized `node` command, the `main` property in the `package.json` will be used.  If neither is found or recognized, then the user will need to set the `dockerRun.command` property of the `docker-run` task used to start the Docker container.
 
-Some Node.js application frameworks include CLIs for managing the application and used to start the application in the `start` script, which obscure the underlying `node` commands. In these cases, the Docker extension cannot infer the start command and the user must configure it manually.
+Some Node.js application frameworks include CLIs for managing the application and are used to start the application in the `start` script, which obscure the underlying `node` commands. In these cases, the Docker extension cannot infer the start command and it must be configured explicitly.
```

Comment:
For actions required of the user, I would call attention to those by using "you" : "you must explicitly configure the start command".

Reply:
Updated.